ansible_collections/arista/avd/extensions/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```
@@ -5265,12 +5265,12 @@ ip nat synchronization
 ip ftp client source-interface Loopback0
 ip ftp client source-interface Management0 vrf MGMT
 ip ftp client source-interface Ethernet10 vrf abc
-ip http client local-interface Loopback0 vrf default
-ip http client local-interface Management0 vrf MGMT
 ip http client local-interface Ethernet10
-ip ssh client source-interface Ethernet10
-ip ssh client source-interface Loopback0 vrf default
+ip http client local-interface Management0 vrf MGMT
+ip http client local-interface Loopback0 vrf aaa
+ip ssh client source-interface ethernet10
```

Let's fix the inpu in moleulce for this
```suggestion
ip ssh client source-interface Ethernet10
```